examples/data_pipeline/pipelines.py

```diff
@@ -5,9 +5,7 @@
     PretrainedSPVocab,
```

Comment:
We should check if there is a need to update the performance numbers in the README file.

On that note, it should be easier to read if we aggregate those benchmark numbers into a table and also add the disclaimer that these numbers are for our development reference only.

Reply:
Done.